docs/hyperion/user-guide/global-configurations/sso-login.md

```diff
@@ -34,7 +34,7 @@ Only single SSO login configuration can be active at one time. Whenever you crea
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
```

Comment:
Except for the domain substring, URL and redirectURI should be the same as in the screenshots.